README.md

```diff
@@ -74,13 +74,13 @@ to do this manually the first time while running Puma. After registering, you ca
 with the code below:
 
 ```python
-from puma.apps.android.whatsapp.whatsapp import WhatsappActions
+from puma.apps.android.whatsapp.whatsapp import WhatsApp
```

Comment:
The WhatsApp examples in the `README.md` were outdated. Decided to update these in this branch as well.